examples/tenant_tutorial/tenant_tutorial/urls_public.py

```diff
@@ -1,6 +1,6 @@
-from django.conf.urls import patterns, url
```

Comment:
But this library has not ended support for Django 1.7 or lower so there can't be any breaking changes that make it stop working for lower versions.